shared/metrics/src/main/java/io/pravega/shared/metrics/OpStatsLoggerImpl.java

```diff
@@ -51,6 +51,7 @@ public void close() {
     }
 
     @Override
+    @SuppressWarnings("deprecation") // deprecated since Java 10
```

Comment:
I'm actually not sure we should remove these warnings. Ideally we should act on them by moving to the new API. Maybe leave them in place until we do.